external_libraries/mpi_python/mpi_python.h

```diff
@@ -1,12 +1,27 @@
-#ifndef KRATOS_MPI_PYTHON_H
-#define KRATOS_MPI_PYTHON_H
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Jordi Cotela
+//                   Michael Andre
+//                   Philipp Bucher
+//
+
+// System includes
+#include <vector>
+#include "mpi.h"
 
-#include <stdio.h>
-#include <iostream>
+// External includes
 
-#include "mpi.h"
-#include <pybind11/pybind11.h>
+// Project includes
 
+#ifndef KRATOS_MPI_PYTHON_H
+#define KRATOS_MPI_PYTHON_H
```

Comment:
Just curious, why did you move the header guard here?

Reply:
I added the default header in those files (was not there before) therefore it shows up as changed

Reply:
I think @msandre  is asking why the header guard does not go at the beginning of the file, before the includes.

(i also wonder the same)

Reply:
Ah now I understand, thx for clarifying

Will change it